packages/@uppy/aws-s3/src/index.js

```diff
@@ -15,8 +15,8 @@
  * will expire before some files can be uploaded.
  *
  * The long-term solution to this problem is to change the upload pipeline so that files
- * can be sent to the next step individually. That requires a breakig change, so it is
- * planned for Uppy v2.
+ * can be sent to the next step individually. That requires a breaking change, so it is
+ * planned for Uppy v4.
```

Comment:
@Murderlon Should it just say `some future version`?

Reply:
Yes, that might be better. Thanks for also updating the comment!